pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -1,29 +1,60 @@
-{ lib, buildPythonApplication, fetchPypi
-, requests, fasteners, pyyaml, pyjwt, colorama, patch
-, bottle, pluginbase, six, distro11, pylint, node-semver2
-, future, pygments, mccabe
-, fetchpatch
-}:
-
-buildPythonApplication rec {
-  version = "1.1.1";
+{ lib, fetchpatch, python }:
+
+let newPython = python.override {
+  packageOverrides = self: super: {
+    distro = super.distro.overridePythonAttrs (oldAttrs: rec {
+      version = "1.1.0";
```

Comment:
Please remove the `name` attribute from distro's and node-semver's `default.nix`. Otherwise, you'd have to override it here too.